client/src/components/styles/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
```

Comment:
Nice job with the styled components! I had my styled components in their own files depending on the component (nearly each component had its own styled component file) but I like having them all in one file like this kind of like a style.css - I'll probably refactor mine to do this since its easy to have all styled components in one place!

Reply:
I actually refactored mine to be like yours and put them in a file that is dedicated to the component its styling.  But I guess if you did it like this it would be just like css.